src/dlgPackageExporter.cpp

```diff
@@ -170,6 +181,39 @@ bool dlgPackageExporter::writeFileToZip(const QString& archiveFileName, const QS
     return true;
 }
 
+void dlgPackageExporter::slot_addDependency()
+{
+    if (input->Dependencies->currentText().isEmpty()) {
+        return;
+    }
+    input->Dependencies->addItem(input->Dependencies->currentText());
+    input->Dependencies->clearEditText();
+}
+
+void dlgPackageExporter::slot_removeDependency()
+{
+    if (input->Dependencies->currentText().isEmpty()) {
+        return;
+    }
+    input->Dependencies->removeItem(input->Dependencies->currentIndex());
+    input->Dependencies->clearEditText();
+}
+
+bool dlgPackageExporter::eventFilter(QObject* obj, QEvent* evt)
```

Comment:
I think you need to test the `obj` value to be certain it applies to the right *object* - possibly that would be the `input` one?
:man_shrugging: